### PR TITLE
Lint ERB files

### PR DESCRIPTION
**Pull request by cjdenio**
Best reviewed by commits. Most of these changes were applied automatically by erb-lint.

Fixes #2962

### CODE REVIEW

**Comment by garyhtou** [comment]
Can we set up a script that will run both erb_lint and rubocop?

File: app/services/flavor_text_service.rb [hunk 1]
```diff
@@ -35,18 +35,18 @@ def development_flavor_texts
   def birthday_flavor_texts
     [
       "Happy birthday! #{['🎂', '🎈', '🎉', '🥳'].sample}",
-      "<a href='https://www.youtube.com/watch?v=XNV2EyC1NrQ' target='_blank' style='color: inherit'>happy birthday</a>",
+      "<a href='https://www.youtube.com/watch?v=XNV2EyC1NrQ' target='_blank' style='color: inherit'>happy birthday</a>".html_safe,
```
Comment (maxwofford) [inline app/services/flavor_text_service.rb]: Is this intentionally added? If I'm reading this right won't this add the literal `<a>` text to the page (not render as an html link)?

Comment (cjdenio) [reply app/services/flavor_text_service.rb]: This is intentional. Rails will escape the flavor text without `.html_safe`, rendering a literal `<a></a>` (previously there was a blanket `.html_safe` on all flavor text)

**Review by maxwofford — APPROVED** [review]
Thanks for adding this!

**Pull request merged**